js/app/test/kitchen_sink.spec.ts

```diff
@@ -262,5 +95,10 @@ test("test outputs", async ({ page }) => {
 
 	const audio = await page.locator("audio").nth(0);
 	const audio_data = await audio.getAttribute("src");
-	await expect(audio_data).toEqual(BASE64_AUDIO);
+
+	await expect(
+		audio_data?.endsWith(
+			"gradio/92aff6ffe140da201a4a94cb3c3b9e1ff0b2a25a/cantina.wav"
+		)
+	).toBeTruthy();
```

Comment:
The paths generated by gradio are different on different machines but I think the end of the path should be consistent.

Reply:
Yes, they should. In this path:

* `gradio/`comes from the global temp directory set by the GRADIO_TEMP_DIR env variable (defaults to `/tmp/gradio`)
* `92aff6ffe140da201a4a94cb3c3b9e1ff0b2a25a` is based on the hash of the file
* `cantina.wav` is the name of the file